REVIEWING.md

```diff
@@ -0,0 +1,84 @@
+- Internal changes not visible/applicable to users (e.g. refactoring, private methods, etc.) are better suited for commit messages than the changelog.
+
+## Public vs. private API
+
+Be aware of what type of API changes are being made. scikit-bio uses the following conventions to distinguish public vs. private API:
+
+**Public API:** API with an import path that doesn't contain leading underscores in any submodule/subpackage/object names. Users and scikit-bio devs can use public API. Example: `skbio.sequence.Sequence`, `skbio.stats.subsample`
+
+**Package-private API:** API with an import path containing a leading underscore in at least one submodule/subpackage. Users should not import package-private API. Package-private API can be imported anywhere *within* the scikit-bio package. Example: `skbio.util._misc.chunk_str`, `skbio.util._testing.ReallyEqualMixin`
+
+**Private API:** API with object name starting with an underscore. Users should not import private API. Private API should only be imported and used in the module where it is defined. It should not be imported in other parts of the scikit-bio package. Example: `skbio.util._testing._normalize_signs`, `skbio.stats.composition._gram_schmidt_basis`
+
+- Prefer defining private APIs and only promote things to the public API that users need access to.
+- Private/package-private API does not need to be decorated with `@experimental`/`@stable`/`@deprecated`, only public API.
+- Within scikit-bio, prefer *using* public APIs defined in the package even if private APIs offer the same functionality.
+
+## API stability
+
+Refer to scikit-bio's [API stability docs](http://scikit-bio.org/docs/latest/user/api_stability.html) for the current API lifecycle.
+
+- Prefer making new APIs experimental. This allows for changes to the API without needing to deprecate it first.
+- Try to avoid changing stable API if at all possible. If a change is necessary, deprecate the API and remove after 2+ minor releases.
+- Under extreme circumstances, a stable API can be changed without deprecation warning to users. This has happened in the past to fix bugs that required an API change. If this happens, discuss with other devs before committing to the change and document this change extensively in the changelog.
+- Always document stable/experimental API changes and any deprecated APIs in the changelog.
+
+## Integration/consistency with existing API
+
+When reviewing API changes/additions, look at them in the context of the rest of the codebase and its existing APIs. For example, are there existing parameter names that could be reused for consistency/predictability? Does the new API (or changed API) compose with other relevant APIs? Example: `skbio.stats.distance.anosim` uses a `permutations` parameter, so a new nonparametric function would want to use this name over something like `n` or `num_permutations`.
+
+## Licensing and attribution
```

Comment:
sure, but there really isn't an ordering to this doc (the checklist needs to be visited in its entirety)